src/styles/Signup.css

```diff
@@ -0,0 +1,79 @@
+.Account {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  height: 100vh;
+  min-width: 700px;
+}
+
+.Account-Content {
+  display: flex;
+  height: calc(100% - 100px);
+  justify-content: center;
+  margin-top: 100px;
+}
+
+.Account-Panel {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  justify-content: center;
+  text-align: left;
+  background: white;
+  box-shadow: 4px 4px 10px #212121;
+  width: 600px;
+  height: 1360px;
```

Comment:
Probably good to add another 60px to the height in Signup and Login for the panels, so the button height is not minimised.

Reply:
will make it consistent